### PR TITLE
Ana/3074 add previous next in proposal page

### LINKED ISSUE #3074: Add previous/next in proposal page to improve proposals navigation
**Is your feature request related to a problem? Please describe.**
There is not an intuitive way to navigate throught proposals. 

**Describe the solution you'd like**
Would be useful to add previous/next arrows linked to previous/next proposals in `PageProposal.vue`

**Describe alternatives you've considered**
Actually none.

**Additional context**

Could be something like:

image

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3074 

**Description:**

In this PR I added the navigation within the proposals described in the issue #3074 

image

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [x] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/governance/PageProposal.vue [hunk 1]
```diff
@@ -261,12 +301,38 @@ export default {
     },
     onDeposit() {
       this.$refs.modalDeposit.open()
+    },
+    getProposalIndex(sign) {
+      let proposalsObj = this.proposals.proposals
+      let proposalsArr = Object.keys(proposalsObj).map(key => proposalsObj[key])
```
Comment (colw) [inline src/components/governance/PageProposal.vue]: I believe `Object.values(proposalsObj)` can help you out here 👍

File: src/components/governance/PageProposal.vue [hunk 2]
```diff
@@ -167,6 +191,22 @@ export default {
     proposal() {
       return this.proposals.proposals[this.proposalId]
     },
+    getNextProposalId() {
+      let id = this.getProposalIndex(`+1`)
+      if (id !== undefined) {
+        return id
+      } else {
+        return this.proposalId
```
Comment (faboweb) [inline src/components/governance/PageProposal.vue]: you could just return `id` and check in the view `v-show = "getPrevProposalId"`
 `

Comment (faboweb) [reply src/components/governance/PageProposal.vue]: would reduce lines of code by a bunch

**Comment by jbibla** [comment]
i like how this works. nice design @mariopino! nice implementation @Bitcoinera! 

can we add this to the block explorer as well 😄?

File: src/components/governance/PageProposal.vue [hunk 3]
```diff
@@ -293,13 +271,103 @@ export default {
...
+    }
+  },
+  apollo: {
+    proposals: {
+      query() {
+        /* istanbul ignore next */
+        return gql`
+          query proposals($networkId: String!) {
+            proposals(networkId: $networkId) {
+              id
```
Comment (faboweb) [inline src/components/governance/PageProposal.vue]: we don't need most of the data. I think just querying for id is enough?

**Pull request merged**